src/ssh-keygen.ts

```diff
@@ -0,0 +1,444 @@
+  options: [
+    {
+      name: "-A",
+      description:
+        "For each of the key types (rsa, dsa, ecdsa and ed25519) for which host keys do not exist, generate the host keys with the default key file path, an empty passphrase, default bits for the key type, and default comment",
+      exclusiveOn: allOptions.filter(
+        (option) => option !== "-a" && option !== "-f"
+      ),
+    },
+    {
+      name: "-a",
+      description:
+        "When saving a private key, this option specifies the number of KDF",
+      args: {
+        name: "rounds",
+        description: "Number of rounds being used",
+        default: "16",
+        isOptional: true,
+        suggestions: ["16"],
+      },
+    },
+    {
+      name: "-B",
+      description:
+        "Show the bubblebabble digest of specified private or public key file",
+      exclusiveOn: allOptions.filter((option) => option !== "-f"),
+    },
+    {
+      name: "-b",
+      description: "Specifies the number of bits in the key to create",
+      args: {
+        name: "bits",
+        description: "Number of bits in the key",
+        default: "3072",
+        suggestions: ["3072"],
+      },
+    },
+    {
+      name: "-C",
+      description: "Provides a new comment",
+      args: {
+        name: "comment",
+        description: "New comment value",
+      },
+    },
+    {
+      name: "-c",
+      description:
+        "Requests changing the comment in the private and public key files",
+      exclusiveOn: allOptions.filter(
+        (option) =>
+          option !== "-a" &&
+          option !== "-C" &&
+          option !== "-f" &&
+          option !== "-P"
+      ),
+    },
+    {
+      name: "-D",
+      description: "Download the public keys provided by the PKCS#11",
+      args: {
+        name: "pkcs11",
+        description: "PKCS#11 public keys",
+      },
+      exclusiveOn: allOptions,
+    },
+    {
+      name: "-E",
+      description: "Specifies the hash algorithm used",
+      args: {
+        name: "fingerprint_hash",
+        description: "Hash algorithm value",
+        default: "sha256",
+        suggestions: [{ name: "md5" }, { name: "sha256" }],
+      },
+    },
+    {
+      name: "-e",
+      description: "Read a OpenSSH key file and print to stdout",
+      exclusiveOn: allOptions.filter(
+        (option) => option !== "-f" && option !== "-m"
+      ),
+    },
+    {
+      name: "-F",
+      description:
+        "Search for the specified hostname (with optional port number)",
+      args: {
+        name: "hostname",
+        description: "Hostname with optional port number",
+      },
+      exclusiveOn: allOptions.filter(
+        (option) => option !== "-l" && option !== "-v" && option !== "-f"
+      ),
+    },
+    {
+      name: "-f",
+      description: "Specifies the filename of the key file",
+      args: {
+        name: "filename",
+        description: "Filename of the key file",
+        template: "filepaths",
```

Comment:
This is an argument indicating a new file item which will be created so the filepaths template is not sufficient to get the best completions
```suggestion
        template: "filepaths",
        suggestCurrentToken: true,
```